README.md

```diff
@@ -1 +1,35 @@
 # krux_s3
+
+`krux_s3` is a library that provides wrapper functions for common S3 usage. It uses `krux_boto` to connect to AWS S3.
+
+## Warning
+
+In the current version, `krux_s3.s3.S3` is only compatible with `krux_boto.boto.Boto` object. Passing other objects, such as `krux_boto.boto.Boto3`, will cause an exception.
```

Comment:
I think we need a couple more .s3 and .boto added on to the end of these.  Maybe six or seven more? ;)